src/integration/pages/storyPage/mundo/__snapshots__/amp.test.js.snap

```diff
@@ -87,20 +87,13 @@ Object {
 
 exports[`AMP Story Page Analytics ATI 1`] = `
 Object {
-  "requests": Object {
-    "base": "https://logws1363.ati-host.net?",
-    "pageview": "\${base}s=598343&s2=62&p=world::mundo.world.story.51266689.page&r=\${screenWidth}x\${screenHeight}x\${screenColorDepth}&re=\${availableScreenWidth}x\${availableScreenHeight}&hl=\${timestamp}&lng=\${browserLanguage}&x1=[urn:bbc:cps:faf2993a-dcc0-0746-81a6-a6aca4cd59b1]&x2=[amp]&x3=[news-mundo]&x4=[es]&x5=[\${sourceUrl}]&x6=[\${documentReferrer}]&x7=[article]&x8=[simorgh]&x9=[Brexit:+qué+cambiará+para+visitar,+trabajar+y+estudiar+en+Reino+Unido+tras+la+salida+del+país+de+la+Unión+Europea+-+BBC+News+Mundo]&x11=[2020-01-30T06:35:49.000Z]&x12=[2020-01-31T23:08:08.000Z]&x13=[Europe~United+Kingdom~Culture~European+Union~Economics~Brexit]&x14=[0b0bfd5a-81b8-43fa-91fc-57f1fcd24487~2e91364c-5c77-4660-b76e-d76202785e64~6a73afa3-ea6b-45c1-80bb-49060b99f864~9a18805c-fb88-40ba-9763-bcc121d5d55c~ca170ae3-99c1-48db-9b67-2866f85e7342~f94293fd-ff83-400e-8bc0-89ec01930fce]&x16=[WS%20-%20Give%20me%20perspective]&x17=[Explainer]&ref=\${documentReferrer}",
-  },
-  "transport": Object {
-    "beacon": false,
-    "image": true,
-    "xhrpost": false,
-  },
-  "triggers": Object {
-    "trackPageview": Object {
-      "on": "visible",
-      "request": "pageview",
-    },
+  "vars": Object {
```

Comment:
Should these snapshots be changing? Looks like we're replacing ATI with Chartbeat here, when they should both be present, surely?

Reply:
Fixed [here](https://github.com/bbc/simorgh/pull/6747/commits/63028a6b1988d245bb0e707e14e51dfdd9ba2978?file-filters%5B%5D=.jsx)